mapflow/mapflow.py

```diff
@@ -151,12 +150,12 @@ def __init__(self, iface) -> None:
         self.processings = []
 
         # Clear previous temp dir, as it is not cleared automatically in exit from QGis
-        previous_temp_dir = self.settings.value("tempdir", None)
+        previous_temp_dir = self.settings.value("temp_dir", None)
```

Comment:
I assume something else uses the 'tempdir' setting when loading QGIS. Because search for text "tempdir" or "tempfile" in a mapflow folder didn't gave any results, but somehow it still creates at the start of QGIS.

My solution for this: use "temp_dir" as a setting key, that way everything works as intended, files are being deleted and we get naming consistancy (since our varaibles are called "temp_dir" as well)